### ISSUE TITLE
Global styles: base font and scale

### ISSUE THREAD

**Issue by karmatosed** [WordPress/gutenberg#21030]
Being able to easily adjust a scale of typography opens up some powerful combinations of design. There could be 2 potential ways of doing this.

## Declare a base font

![image](https://user-images.githubusercontent.com/253067/77113831-6179e700-6a23-11ea-92f4-f8e076694784.png)

## Pull base from text
The other option is to have the text size be the base and just scale up from it, simply showing a scale and in conjunction with #21028 show that size.

![image](https://user-images.githubusercontent.com/253067/77113881-7ce4f200-6a23-11ea-8107-d0006d4c289a.png)

## Wider picture
In the context of global styles this could look like the following picture:

<img width="1260" alt="all" src="https://user-images.githubusercontent.com/253067/77114035-c5041480-6a23-11ea-9f9a-307bf962b438.png">

## Feedback
As it would be great to move this to development, deciding which approach would be a great step here to continue forward.

**Comment by jasmussen** [comment]
Really nice mockups.

As I looked at the GIF of the delicious work in progress in https://github.com/WordPress/gutenberg/pull/20530#discussion_r400971533, however, I realized that headings in my themes often diverge wildly from a typescale, whether it be through font sizes, or even font weights and styles, perhaps even fonts. Locking it to body and header feels like it might be a simpler user interface up front compared to showing options for headings 1 through 6 — but the latter ultimately seems like the way that will frustrate less.
